lblod/frontend-embeddable-notule-editor
# Update editor packages and related dependencies

## Overview
- Update of `@lblod/ember-rdfa-editor` to version 10.0.0. Note: this dependency update also includes dependency updates of other required packages.
- Update of `@lblod/ember-rdfa-editor-lblod-plugins` to version 22.0.1
  * Addition of the new article `structure` node.
  * Removal of the classic `decision` prosemirror-nodes. These were no longer required and are now replaced by more generic RDFa nodes.

### How to test/reproduce
- Start the test-app
- Notice that nothing breaks (all deps should load correctly)
- The new decision article structure nodes should work as expected

### Checks PR readiness
